reana_commons/config.py

```diff
@@ -149,7 +149,7 @@
 
 WORKFLOW_RUNTIME_USER_GID = os.getenv(
     'WORKFLOW_RUNTIME_USER_GID',
-    0)
+    1000)
```

Comment:
Hmm, this bit may need some considerations... Perhaps something later?  We already mention GID=0 in the docs following OpenShift, the experiment containers already started to use GID=0 in addition to their groups (see below), so changing now to 1000 may not be advantageous... Plus we'd have to retest writing to shared PVC.

```console
$ docker run --rm -i -t reanahub/reana-demo-atlas-recast-eventselection /bin/bash
[bash][atlas]:build > id
uid=500(atlas) gid=500(atlas) groups=500(atlas),0(root)
$ docker run --rm -i -t cmsopendata/cmssw_5_3_32 /bin/zsh
Setting up CMSSW_5_3_32
CMSSW should now be available.
[18:21:36] cmsusr@fb0a7adc97f7 ~/CMSSW_5_3_32/src $ id
uid=1000(cmsusr) gid=1000(cmsusr) groups=1000(cmsusr),0(root)
```

Reply:
(BTW, the "default docker group" is 995 on my Arch Linux, the 1000 is usually reserved for the first non-root user...)

Reply:
It works with 0 as well, I will revert the change.